src/handle.ml

```diff
@@ -209,7 +218,7 @@ let handle_cmd : sig_state -> p_command -> sig_state * proof_data option =
       (* Obtaining the implicitness of arguments. *)
       let impl = Scope.get_implicitness a in
       (* Scoping the type (statement) of the theorem. *)
-      let a = scope_basic a in
+      let a = scope_basic Public a in (* Theorems are public *)
```

Comment:
No, private theorems should be allowed too. You could want to have auxiliary lemmas that you do not want to export.